cdap-docs/developers-manual/source/building-blocks/datasets/index.rst

```diff
@@ -84,6 +84,23 @@ interface or by extending existing Dataset types. See the
 :ref:`Purchase Example<examples-purchase>` for an implementation of a Custom Dataset.
 For more details, refer to :ref:`Custom Datasets. <custom-datasets>`
 
+.. rubric::  Dataset Time-To-Live (TTL)
+
+Datasets, like :ref:`Streams <streams>`, can have a Time-To-Live (TTL) property that
+governs how long data will be persisted in a specific Dataset.
+
+When you create a Dataset, you can configure its TTL as part of the creation::
+
+  public void configure() {
+      createDataset("myCounters", Table.class, 
+                    DatasetProperties.builder().add(Table.PROPERTY_TTL, 
+                                                    "<age in milliseconds>").build());
+      ...
+  }
+
+The default TTL for all Datasets is infinite, meaning that data will never expire. The TTL
```

Comment:
We should mention explicitly that the time unit is milliseconds.  Something like: "TTL is configured as the maximum age in milliseconds for data that should be retained."